### PR TITLE
JS Style Guide: Fix full file closures

### PR DESCRIPTION
**Pull request by scottgonzalez**

### CODE REVIEW

**Comment by jzaefferer** [comment]
:+1: 

Could use the opportunity to add an example for node, e.g. a `Gruntfile.js`, something like this:

```js
var path = require( "path" );

module.exports = function( grunt ) {

grunt.registerTask( "custom task", function() {
	// task impl
} );

};
```

**Comment by scottgonzalez** [comment]
Added.

File: pages/style-guide/js.md [hunk 1]
```diff
@@ -249,35 +249,42 @@ elements
...
+( function( $ ) {
 
-// this doesn't get indented
+// This doesn't get indented
 
-})( jQuery );
+} )( jQuery );
 ```
 
+```
```
Comment (jzaefferer) [inline pages/style-guide/js.md]: Missing `js` language instruction

**Comment by arthurvr** [comment]
Looking good :+1:

File: pages/style-guide/js.md [hunk 2]
```diff
@@ -249,35 +249,42 @@ elements
...
+} )( jQuery );
 ```
 
+```js
+module.exports = function( grunt ) {
+
+// This doesn't get indented
+
+};
+
```
Comment (jzaefferer) [inline pages/style-guide/js.md]: Also missing closing backticks, messing up the highlighting of the remaining file...

**Comment by jzaefferer** [comment]
Good to land.

**Pull request closed without merging**